subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -52,7 +52,8 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
         "1.14",
         "1.15.1",
         "1.15.2",
-        "1.16"
+        "1.16",
+        "2.0"
```

Comment:
I guess the `@Ignore` from this test should be removed. I also think that only `2.0` is supported and no other version.

Reply:
Fixed, I removed the `GRACEFULLY_UNSUPPORTED` list and test, as we now seem to print a warning for any version prior to 2.0